roles/internal/crayfish/tasks/install.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+- name: Install requisite package repositories
+  apt_repository:
+    repo: 'ppa:lyrasis/imagemagick-jp2'
+
+- name: Install requisite packages
+  apt:
+    name: "{{item}}"
+    state: present
+  with_items:
+    - tesseract-ocr
+    - imagemagick
+
+- name: Install crayfish code
+  git:
+    repo: https://github.com/Islandora-CLAW/Crayfish.git
+    dest: "/var/www/html/Crayfish"
+    version: "{{ crayfish_version_tag }}"
+    
+- name: Build crayfish code including dependencies
+  composer:
+    command: install
+    working_dir: /var/www/html/Crayfish/{{item}}
+  with_items: "{{ crayfish_services }}"
+    
+- name: Configure crayfish code
+  template:
+    src: "{{item}}.config.yaml.j2"
+    dest: "/var/www/html/Crayfish/{{item}}/cfg/config.yaml"
+  with_items: "{{ crayfish_services }}"
+    
+- name: Install auth config
+  template:
+    src: "syn-settings.xml.jp2"
+    dest: "/var/www/html/Crayfish/{{item}}/syn-settings.xml"
+  with_items: "{{ crayfish_services }}"
+
+- name: Create Gemini db
+  mysql_db:
+    name: "{{ gemini_db_options.dbname }}"
+    state: present
```

Comment:
In order to check the `changed` property, you have to register a variable. So this would have to be: 
```yaml
- name: Create Gemini db
  mysql_db:
    name: "{{ gemini_db_options.dbname }}"
    state: present
  register: gemini_db_exists
```

Reply:
Got it. That's what creates the ability to check changed after this step, right?

Reply:
yup!